### PR TITLE
Ipynb update

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

update notebooks

## Before submitting
- [ ] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [ ] Did you make sure to update the documentation with your changes? (if necessary)
- [ ] Did you write any new necessary tests? (not for typos and docs)
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Review by tchaton — APPROVED** [review]
LGTM !

**Comment by Borda** [comment]
see docs in https://pytorch-lightning.readthedocs.io/en/ipynb-update/
wait for one more update from https://dev.azure.com/PytorchLightning/Tutorials/_build/results?buildId=13491&view=logs&j=4d809e3c-18a0-5a8d-e0ac-4c5682e3f11b

File: .gitmodules [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[submodule "notebooks"]
-	path = notebooks
+[submodule "_notebooks"]
+	path = _notebooks
```
Comment (awaelchli) [inline .gitmodules:2]: what does this do?

**Review by awaelchli — APPROVED** [review]

File: .gitmodules (hunk 1 above)
Comment (Borda) [reply .gitmodules:2]: git does and eventually some IDEs are not happy when you switch between master (or PRs created from before refactored) and release branch as once it is a folder other time it is a reference to submodule... also a little visual diff that it is not a normal folder :rabbit:

**Review by justusschock — APPROVED** [review]

File: docs/source/index.rst [hunk 2]
```diff
@@ -57,10 +57,11 @@ PyTorch Lightning Documentation
...
    From PyTorch to PyTorch Lightning [Video] <https://www.youtube.com/watch?v=QHww1JH7IDU>
-   notebooks/starters/mnist-hello-world.ipynb
-   notebooks/starters/datamodules.ipynb
-   notebooks/starters/cifar10-baseline.ipynb
-   notebooks/starters/basic-gan.ipynb
+   notebooks/lightning_examples/mnist-hello-world.ipynb
+   notebooks/lightning_examples/datamodules.ipynb
+   notebooks/lightning_examples/cifar10-baseline.ipynb
+   notebooks/lightning_examples/basic-gan.ipynb
+   notebooks/lightning_examples/augmentation_kornia.ipynb
```
Comment (Borda) [inline docs/source/index.rst:64]: need to name explicitly the notebooks, @justusschock @edgarriba

**Review by edgarriba — APPROVED** [review]

**Comment by carmocca** [comment]
@Borda milestone?

**Pull request merged**